contestantA/src/main/java/edu/asu/diging/contestantA/services/Play.java

```diff
@@ -0,0 +1,123 @@
+package edu.asu.diging.contestantA.services;
+
+import java.util.ArrayList;
+import java.util.Random;
+
+import org.apache.log4j.Logger;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.kafka.annotation.KafkaListener;
+import org.springframework.stereotype.Service;
+
+@Service
+public class Play {
+	@Autowired
+	private Sender send;
+	final static Logger logger = Logger.getLogger(Play.class);
+	boolean started = false;
+	private String tossValue;
+	boolean won = false;
+	int current = -1;
+	ArrayList<String> playerOne;
+	ArrayList<String> playerTwo;
+	Toss toss;
+
+	Play() {
+		playerOne = new ArrayList<String>();
+		playerTwo = new ArrayList<String>();
+		Random rand = new Random();
+		current = rand.nextInt(100);
+	}
+
+	public int getCurrentNumber() {
+		return current;
+	}
+
+	public int getHalfCurrentNumber() {
+		return current / 2;
+	}
+
+	public ArrayList<String> getListOfANumbers() {
+		return playerOne;
+
+	}
+
+	public ArrayList<String> getListOfBNumbers() {
+		return playerTwo;
+
+	}
+
+	public void setPlayerOne(String str) {
+		playerOne.add(str);
+	}
+
+	public void setPlayerTwo(String str) {
+		playerTwo.add(str);
+	}
```

Comment:
the last two methods are not setting anything. Why are they called "setPlayer..."?